modules/apps/commerce/commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/pricing/test/CommercePriceListLowestDiscoveryTest.java

```diff
@@ -113,6 +115,12 @@ public void setUp() throws Exception {
 			_group.getGroupId(), _commerceCurrency.getCode());
 	}
 
+	@After
+	public void tearDown() throws Exception {
```

Comment:
@riccardo-alberti @marco-leo shouldn't it be better to have base class that would hold this copy/pasted code?? That is why we have OOP